### PR TITLE
Issue #435: README.md exmaple does not work

### PR DESCRIPTION
**Pull request by iStryker**

### CODE REVIEW

**Comment by e0ipso** [comment]
Many thanks for the contribution!

File: README.md [hunk 1]
```diff
@@ -63,12 +63,12 @@ dependencies[] = restful
  }
 ```
 
-####restful\_custom/plugins/restful/myplugin.inc
+####restful\_custom/plugins/restful/thenameoftheplugin.inc
```
Comment (amitaibu) [inline README.md:66]: maybe just change this to `restful\_custom/plugins/restful/articles.inc`, and keep the comment about the plugin name == the `name` property?

Comment (iStryker) [inline README.md:66]: I was thinking along the same lines, however articles.inc is a little bit confusing with 'resource' and 'name' being the same.  It doe not emphasize that the name of the file has to be the same as the name of the plugin.   

Maybe changing it to articles.inc and having 'The name of your plugin must match your file' comment I added is good enough for some. Personally, I would not notice it right away, however once I received the thrown error, I would re-read the instruction and noticed the comment right away. 

If you are not happy with 'thenameoftheplugin' you could change it to your liking, like 'plugin_name'

**Comment by amitaibu** [comment]
Seems #446 already took care of this. Re-open if you think it's needed.

**Pull request closed without merging**